client/app/index.html

```diff
@@ -246,6 +246,12 @@
 
         config.previewSubjectFilterKey = 'patopic';
 
+        //date formats for publish/embargo entry
+        config.view = {
+            dateformat: 'D/M/YYYY',
```

Comment:
The model formats are

`model: {
       dateformat: 'DD/MM/YYYY',
       timeformat: 'HH:mm:ss'
},`